specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2018-02-14/keyvault.json

```diff
@@ -0,0 +1,1359 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-02-14-preview",
```

Comment:
>2018-02-14-preview [](start = 16, length = 18)

Folder should have the same name as version here

Reply:
Is this required?  I thought having preview in both the path and the folder name would be redundant.  I followed the pattern I used in the data-plane:

https://github.com/Azure/azure-rest-api-specs/tree/master/specification/keyvault/data-plane/Microsoft.KeyVault/preview

Reply:
Yes, it is required